src/main/java/com/google/starfish/services/GenericService.java

```diff
@@ -0,0 +1,57 @@
+package com.google.starfish.services;
+
+import java.sql.ResultSet;
+import javax.sql.DataSource;
+import java.sql.PreparedStatement;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+/** Enum that holds starfishDB sql tables as strings */
+enum Table {
+  USERS("users"), NOTES("notes"), LABELS("labels"), FAVORITE_NOTES("favorite_notes"), MISC_LABELS("misc_note_labels");
+
+  private String sqlTable;
+
+  public String getSqlTable() {
+    return this.sqlTable;
+  }
+
+  private Table(String sqlTable) {
+    this.sqlTable = sqlTable;
+  }
+}
+
+/** Generic Service class that provides the functionality to get and delete row by id in any table */
+public class GenericService {
```

Comment:
nit: maybe naming it something like `TableService` would be more descriptive

Reply:
I agree, "Generic" sounds very... dare I say, generic.